src/assets/sass/protocol/base/elements/_forms.scss

```diff
@@ -0,0 +1,87 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../../includes/lib';
+
+label {
+    display: block;
+    font-weight: bold;
+    margin: 0 0 .25em;
+    padding: 0 $spacing-sm;
+
+    &.mzp-u-inline {
+        @include bidi(((padding, 0 $spacing-sm 0 0, 0 0 0 $spacing-sm),));
+        display: inline;
+        margin: 0;
+    }
+}
+
+button,
+input,
+select,
+textarea {
+    @include open-sans;
+    @include text-body-md;
+}
+
+textarea {
+    height: auto;
+    max-width: 100%;
+}
+
+textarea,
+input[type='email'],
+input[type='url'],
+input[type='tel'],
+input[type='password'],
+input[type='search'],
+input[type='text'],
+input[type='number'],
+input[type='date'],
+input[type='time'] {
+    @include border-box;
+    background: #fff;
+    border-radius: .25em;
+    border: 1px solid $color-gray-dark;
+    display: inline-block;
+    line-height: 1.25;
+    padding: .25em $spacing-sm;
+
+    &:hover {
+        border-color: #000;
+    }
+
+    &:focus {
+        border-color: #000;
+        box-shadow: 0 0 0 2px rgba(73, 173, 227, .4);
+    }
+
+    &:disabled {
+        background: $color-gray-lighter;
+
+        &:hover,
+        &:focus {
+            border-color: $color-gray-dark;
+        }
+    }
+}
+
+select:focus {
+    box-shadow: 0 0 0 2px rgba(73, 173, 227, .4);
+}
+
+legend {
+    @include text-display-sm;
+}
+
+fieldset li {
+    margin-bottom: .5em;
+}
+
+.mzp-c-fieldnote {
+    @include text-body-sm;
+    color: #333;
```

Comment:
Could we use a color here from our tokens maybe? `$color-gray-dark`?

Reply:
`$color-gray-dark` still wasn't dark enough for my tastes, but we can use it for now and update once we have our new grayscale defined.